package.json

```diff
@@ -13,7 +13,9 @@
     "prepare": "npm run build",
     "test": "mocha --exit --timeout 1m \"./test/unit/**/*-specs.js\"",
     "e2e-test": "mocha --exit --timeout 10m \"./test/functional/**/*-specs.js\"",
-    "bundle": "node ./Scripts/build-webdriveragent.js",
+    "bundle": "npm run bundle:ios && npm run bundle:tv",
```

Comment:
can we run these in parallel?

Reply:
Might be...? Parallel xcodebuild raised some build errors long before, so here runs one by one. Maybe some part such as creating zip can be parallel though

Reply:
Ran the xcodebuild in parallel. Then exit 65 code was raised, so currently it would be safe to run one by one. We'll be able to optimize parallel running except for xcodebuild in the future